### DISCUSSION TITLE
Welcome to cocalc Discussions!

### DISCUSSION THREAD

**Discussion by williamstein** [sagemathinc/cocalc#5069] [General]
This is a test of GitHub discussions for CoCalc.

**Comment by standekansas** [comment]
Does this (the github site) mean you are shutting down Discord?  Is SimPy available on CoCalc, or could it be? If not, is there a simulation module you recommend?
Thanks,
Stan

**Reply by williamstein** [reply]
> Does this (the github site) mean you are shutting down Discord?

We will leave it running indefinitely.  However, don't expect much from it.

> Is SimPy available on CoCalc

Yes, it is installed right now into Sage:

<img width="759" alt="image" src="https://user-images.githubusercontent.com/1276278/202886880-b0ce7b77-2cb8-4baf-96a5-44e61b18b1ee.png">

You can find what is installed and where here: https://cocalc.com/software/python

You can type `sage -python` to the python that is part of sage, incidentally.  Also, just select the Sage python kernel.
